src/sentry/identity/oauth2.py

```diff
@@ -357,7 +357,7 @@ def dispatch(self, request: Request, pipeline) -> HttpResponse:
                 lifecycle.record_failure(
                     "token_exchange_error", extra={"failure_info": ERR_INVALID_STATE}
                 )
-                return pipeline.error(ERR_INVALID_STATE)
+                return pipeline.error(ERR_INVALID_STATE + f"\nError: {error}")
```

Comment:
Very minor nit, but why not just throw the `ERR_INVALID_STATE` string in the fstring string?